server/helpers/verifyEmailMessage.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @description verifyEmailMessage contains the email message template
+ * @param {string} token - The token generated for the user
+ * @returns {object} - contains the mail message template
+ */
+
+const verifyEmailMessage = (token) => {
+  if (token === undefined || token === '') {
+    const err = new Error('token is invalid');
+    err.status = 404;
+    // console.log(err);
```

Comment:
Do we need this here?

Reply:
Yes. I feel there is need to ensure the `token` is not undefined and not an empty string.